pandas/tests/sparse/test_frame.py

```diff
@@ -1132,7 +1132,7 @@ def test_isnotnull(self):
 @pytest.mark.parametrize('dtype', [object, bool, int, float, np.uint16])
```

Comment:
should we just not test this at all (object I mean)? I don't actually have scipy 0.19.0 installed, so can't really test this.